cargo-dist/templates/installer/homebrew.rb.j2

```diff
@@ -30,6 +30,11 @@ class {{ formula_class }} < Formula
   {%- if license %}
   license "{{ license }}"
   {%- endif %}
+  {%- if dependencies|length > 0 %}
+  {% for dep in dependencies %}
+  depends_on "{{ dep }}"
+  {%- endfor %}
+  {%- endif %}
```

Comment:
hmm what does the if-guard eliminate here? An extra newline?

Reply:
Yeah, that's right. I'm placing a newline between the "package definition" stuff that ends with the version and the start of the dependencies. That's Homebrew's own standard.